app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,35 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
```

Comment:
- [ ] Please remove the empty methods present in this file. It doesn't look professional.

Reply:
Hello @emmanuellekamwa If you check my code you'll see that you pointed to a method that has already been removed.